modules/tinymce/src/themes/silver/main/ts/ui/menus/menu/searchable/SearchableMenus.ts

```diff
@@ -0,0 +1,23 @@
+import { ItemTypes } from '@ephox/alloy';
+import { Id } from '@ephox/katamari';
+
+export const searchResultsClass = 'tox-collection--results__js';
+
+// NOTE: this is operating on the the final AlloySpec
+export const augmentWithAria = (item: ItemTypes.ItemSpec): ItemTypes.ItemSpec => {
+  if (item.dom) {
+    return {
```

Comment:
I wonder if this would just be better off doing a `Merger.deepMerge`?

Reply:
I've tended to avoid it for fears of performance issues, but if we still use it in situations where it makes it clearer, then I'm happy to use it.

Reply:
Fair, it's still used in quite a number of places so it shouldn't be an issue but at the same time I'm fine leaving this as-is 🤷